services/motion/builtin/builtin.go

```diff
@@ -329,6 +331,10 @@ func (ms *builtIn) MoveOnGlobe(
 	}
 }
 
+func (ms *builtIn) MoveOnGlobeNew(ctx context.Context, req motion.MoveOnGlobeReq) (string, error) {
```

Comment:
I changed the signature of `MoveOnGlobeNew` to take a `motion.MoveOnGlobeReq` as the number of parameters is already getting unwieldy